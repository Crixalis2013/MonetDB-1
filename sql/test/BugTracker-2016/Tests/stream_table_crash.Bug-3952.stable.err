stderr of test 'stream_table_crash.Bug-3952` in directory 'sql/test/BugTracker-2016` itself:


# 22:03:48 >  
# 22:03:48 >  "mserver5" "--debug=10" "--set" "gdk_nr_threads=0" "--set" "mapi_open=true" "--set" "mapi_port=34898" "--set" "mapi_usock=/var/tmp/mtest-8540/.s.monetdb.34898" "--set" "monet_prompt=" "--forcemito" "--dbpath=/home/niels/scratch/rc-clean/Linux-x86_64/var/MonetDB/mTests_sql_test_BugTracker-2016" "--set" "embedded_r=yes"
# 22:03:48 >  

MAPI  = (monetdb) /var/tmp/mtest-314879/.s.monetdb.36577
QUERY = create stream table temp(t timestamp, room integer, payload decimal(8,2));
ERROR = !syntax error in: "create stream"
CODE  = 42000

# 22:03:49 >  
# 22:03:49 >  "Done."
# 22:03:49 >  

