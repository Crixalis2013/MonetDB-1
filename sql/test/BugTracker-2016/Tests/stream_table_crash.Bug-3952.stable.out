stdout of test 'stream_table_crash.Bug-3952` in directory 'sql/test/BugTracker-2016` itself:


# 22:03:48 >  
# 22:03:48 >  "mserver5" "--debug=10" "--set" "gdk_nr_threads=0" "--set" "mapi_open=true" "--set" "mapi_port=34898" "--set" "mapi_usock=/var/tmp/mtest-8540/.s.monetdb.34898" "--set" "monet_prompt=" "--forcemito" "--dbpath=/home/niels/scratch/rc-clean/Linux-x86_64/var/MonetDB/mTests_sql_test_BugTracker-2016" "--set" "embedded_r=yes"
# 22:03:48 >  

#create table temp(t timestamp, room integer, payload decimal(8,2));
#insert into temp values(timestamp '2016-03-13 08:58:14', 1, 23.4);
[ 1	]
#drop table temp;


# 22:03:49 >  
# 22:03:49 >  "mclient" "-lsql" "-ftest" "-Eutf-8" "-i" "-e" "--host=/var/tmp/mtest-8540" "--port=34898"
# 22:03:49 >  

#create stream table temp(t timestamp, room integer, payload decimal(8,2));

# 22:03:49 >  
# 22:03:49 >  "Done."
# 22:03:49 >  

